rs-e2e/readme.md

```diff
@@ -0,0 +1,57 @@
+# ReportStream Integration Test
+
+The ReportStream Integration Test is a framework meant to add test coverage for the integration between the
+Intermediary and ReportStream. It's scheduled to run daily using the
+[automated-staging-test-run.yml](../../../.github/workflows/automated-staging-test-run.yml) workflow
+
+Information on how to set up the sample files evaluated by the tests can be found [here](../examples/Test/Automated/readme.md)
```

Comment:
Again, if it makes sense to you to use an absolute path from the root of the repo, you can.  And lower in this readme too, if you want.

```suggestion
[automated-staging-test-run.yml](/.github/workflows/automated-staging-test-run.yml) workflow

Information on how to set up the sample files evaluated by the tests can be found [here](/examples/Test/Automated/readme.md)
```